core/src/translate/function.rs

```diff
@@ -142,6 +142,14 @@ pub fn translate_function(sql_function: &SqlFunction) -> Result<Expr> {
         .collect::<Result<Vec<_>>>()?;
 
     match name.as_str() {
+        "CONCAT" => {
+            check_len_range(name, args.len(), 2, 255)?;
```

Comment:
It would be good if the `CONCAT` function works if the length is 1 or more.
Is there a reason there is a 255 count limit?

Reply:
e.g)
```sql
mysql> select concat("Adsf");
+----------------+
| concat("Adsf") |
+----------------+
| Adsf           |
+----------------+
1 row in set (0.00 sec)
```

Reply:
I went with a limit of 2 to 255 because of the major sql databases (postgres or maybe it was MSSQL) had those limit  We can definitely change those limits.   We can go with 1 for the lower limit and no upper limit

Reply:
I have a personal question. (since the database I mainly use is MySQL)
PostgreSQL, why limit the length in MSSQL?
I wonder what the definition for it is

Reply:
I've used mostly MySQL and Server Server (MSSQL).   I've played a bit with postgres just for fun.  
Here's info on SQL Server's implementation of concat: https://docs.microsoft.com/en-us/sql/t-sql/functions/concat-transact-sql?view=sql-server-ver15

They seem to have gone the same route as postgres and just ignore NULL values by converting them to the empty space.

Reply:
I believe the number of args issue has been resolved.  If not, let me know.